src/browser/base/content/browser-sidebar-js.patch

```diff
@@ -165,7 +205,89 @@ index f09e37fc42f7754f7f3687884ead25124c1b2076..e1833bbfd8a8ed7da767349f886d554f
        }
      }
    },
-@@ -573,6 +652,10 @@ var SidebarUI = {
+@@ -485,6 +588,11 @@ var SidebarUI = {
+   },
+ 
+   _loadSidebarExtension(commandID) {
++    if (this.keepTabsActivePref && this.loadedSidebars.includes(commandID)) {
++      console.log("returned");
++      return;
++    }
++    console.log("it went on");
```

Comment:
Also, you still have debug console.log statements here

Reply:
Also left overs from testing which I forgot to wipe.